README.md

```diff
@@ -197,3 +194,17 @@ OpenAPI schema can be verified on localhost using following command:
     OPENAPI_HOST=http://localhost:8000 ./.jenkins/openapi_validator/run_validator.sh
 
 Do not forget to replace `OPENAPI_HOST` with address of running DSO api, if it differs from `http://localhost:8000`.
+
+
+# Limiting number of Datasets available via API
+
+In some cases it might be required to isolate datasets from others on infrastructure level, 
+for example sensitive information should not be published via Public APIs, but public information should be available in private API.
+
+This can be achieved by using `DATASETS_LIST` and `DATASETS_EXCLUDE` environment variables.
+
+In case API should expose only limited set of datasets - use `DATASETS_LIST` to load only provided datasets into memory.
+In case API should expose all datasets with exception to limited subset - use `DATASETS_EXCLUDE` to loxad all datasets except provided into memory.
```

Comment:
"Loaded into memory" suggests datasets are resident in RAM, but they stay on disk, right?

Also, the syntax for these environment variables' values is missing. If I want to make foo and bar available, is that `foo bar`, `foo,bar`, `foo+bar`, ...

Reply:
> "Loaded into memory" suggests datasets are resident in RAM, but they stay on disk, right?

Nope, they are loaded into memory for worker lifespan. Disk/Database usage during API construction is not advised, therefore everything is loaded into memory.

> Also, the syntax for these environment variables' values is missing. If I want to make foo and bar available, is that foo bar, foo,bar, foo+bar, ...
Good one, will add.

Reply:
Ok, I hadn't figured that out yet.

Reply:
Actually, they are loaded in memory. That is, the jsonschema definitions are read on startup of the Django application and converted into on-the-fly Django models in memory.

Reply:
This is confusing :) What is loaded into memory? The entire datasets, or just their schema definitions?

Reply:
Whole API is built in memory and based on Dataset model, which is read only inside Router.

This PR is allowing users to limit datasets on which API is build. However it is easier to say "Datasets loaded into memory" as whole "in memory API" logic is complex and desires whole separate document on mechanics of API construction. (we have ticket for that).

Hope this clears things up, let me know if you want this to be added to current readme.

Reply:
May I suggest an alternative phrasing:

* To expose only a subset of the datasets, use `DATASETS_LIST`.
* To expose all datasets with some exceptions, use `DATASETS_EXCLUDE`.

This avoids discussing implementation details, while also not suggesting that massive amounts of RAM need to be provisioned to get this service running.